markdown_xblock/static/css/pygments.css

```diff
@@ -1,61 +1,61 @@
-.markdown_block .codehilite .hll { background-color: #ffffcc }
```

Comment:
Not sure what this diff is about? Is it intentionally in this commit?

Reply:
Yes, for the `codehilite` to work this will need to match the class in `/static/html/lms.html`.

Reply:
Oh! Of course. Sorry, I didn't catch that.

OK in that case would you mind splitting this commit into two, one that does the `markdown_block` → `markdown_xblock` rename and one that adds the `classes` field?

It can still be one PR, but the two changes should be one commit each.

Reply:
Ack, sure.